R/network.R

```diff
@@ -339,20 +339,30 @@ nearest_node <- function(network, target) {
   nodes[idx]
 }
 
-#' Subset a network keeping the only nodes that intersect a target geometry.
+#' Subset a network keeping the components that intersect a target geometry.
```

Comment:
Maybe a different word than "component" to avoid confusion with (connected) components as connected regions/sub-graphs in a network? Maybe we can call them "elements" instead?

Reply:
You are right, it is probably better to stick to the tidigraph naming of "component" as sub-graph. Will adapt to use "elements" instead!